packages/ibm-products/src/components/Coachmark/Coachmark.tsx

```diff
@@ -207,10 +209,15 @@ export let Coachmark = forwardRef<HTMLElement, CoachmarkProps>(
         setShouldResetPosition(true);
       }
     };
-    const overlayPositionStyle = {
-      top: (positionTune?.y ?? 0) - 16,
-      left: (positionTune?.x ?? 0) - 16,
-    };
+    const overlayPositionStyle = useMemo(() => {
+      const top = (positionTune?.y ?? 0) - 16;
+      const left = (positionTune?.x ?? 0) - 16;
+
+      return {
+        top: `${top}px`,
+        left: `${left}px`,
+      };
+    }, [positionTune]);
```

Comment:
```suggestion
    const overlayPositionStyle =  {
        top: `${(positionTune?.y ?? 0) - 16}px`,
        left: `${(positionTune?.x ?? 0) - 16}px`,
      };
```

No need to add useMemo here, plus it will be removed next React update :)

Reply:
Hey @AlexanderMelox, thanks for reviewing. 

I am using `overlayPositionStyle` as a dependency of `useIsomorphicEffect`, In that case if i run yarn lint, it will throw below lint error

```
 @typescript-eslint/no-unused-vars
  213:11  error  The 'overlayPositionStyle' object makes the dependencies of useIsomorphicEffect Hook (at line 261) change on every render. Move it inside the useIsomorphicEffect callback. Alternatively, wrap the initialization of 'overlayPositionStyle' in its own useMemo() Hook  react-hooks/exhaustive-deps
```